WORKSPACE

```diff
@@ -5,42 +5,20 @@ http_archive(
   name = "build_bazel_rules_nodejs",
   url = "https://github.com/bazelbuild/rules_nodejs/archive/0.15.3.zip",
   strip_prefix = "rules_nodejs-0.15.3",
-  sha256 = "05afbbc13b0b7d5056e412d66c98853978bd46a94bc8e7b71c7fba4349b77eef",
```

Comment:
Note: not recommended to use sha256 with github.com archives since the zip files is generated on the fly by github and the sha256 may change in the future